vue/sbc-common-components/src/util/constants.ts

```diff
@@ -7,6 +7,8 @@ export enum SessionStorageKeys {
   LaunchDarklyFlags = 'LD_FLAGS',
   CurrentAccount = 'CURRENT_ACCOUNT',
   AuthApiUrl = 'AUTH_API_URL',
+  AuthWebUrl = 'AUTH_WEB_URL',
+  StatusApiUrl = 'STATUS_API_URL',
```

Comment:
Where is this used?

Reply:
use in config-helper. I make the code looks like consistence.

Reply:
I see it now 👍